doc/source/missing_data.rst

```diff
@@ -534,7 +520,7 @@ the ``limit_area`` parameter restricts filling to either inside or outside value
 Replacing Generic Values
 ~~~~~~~~~~~~~~~~~~~~~~~~
 Often times we want to replace arbitrary values with other values. The
-``replace`` method in Series/DataFrame provides an efficient yet
+:meth:`~DataFrame.replace` method in Series/DataFrame provides an efficient yet
```

Comment:
Remove "method". Or write out :meth:\`Series.replace\` and :meth:\`DataFrame.replace\`.